monitoring/variables.tf

```diff
@@ -0,0 +1,128 @@
+variable "environment" {}
+variable "project" {}
+
+variable "cluster_name" {}
+
+variable "ecs_service_role" {}
+
+variable "desired_count" {
+  default = "1"
+}
+
+variable "prometheus_port" {
+  default = "9090"
+}
+
+variable "vpc_id" {}
+
+variable "ecs_sg" {}
+
+variable "r53_zone" {}
+
+variable "r53_zone_prefix" {
+  default = ""
+}
+
+variable "cpu_prometheus" {
+  default = "0"
+}
+
+variable "memory_prometheus" {
+  default = "100"
+}
+
+variable "memory_reservation_prometheus" {
+  default = "100"
```

Comment:
Do you think this will be enough (considering our k8s setups use quite a bit more)?

Reply:
Yeah, TBH I have no clue how much it will need. 100M seems very low, but let's start with this anyway and see whether it works or not.

Reply:
We have a reservation of 2.5Gb for Prometheus in k8s 😛